sdk/communication/Azure.Communication.Chat/src/ChatThreadClient.cs

```diff
@@ -598,13 +650,14 @@ async Task<Page<ChatMessageReadReceipt>> FirstPageFunc(int? pageSizeHint)
                     throw;
                 }
             }
-            return PageableHelpers.CreateAsyncEnumerable(FirstPageFunc, null);
+            return PageableHelpers.CreateAsyncEnumerable(FirstPageFunc, NextPageFunc);
         }
 
         /// <summary> Gets read receipts for a thread. </summary>
+        /// <param name="skip"> Skips chat message read receipts up to a specified position in response. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
-        public virtual Pageable<ChatMessageReadReceipt> GetReadReceipts(CancellationToken cancellationToken = default)
+        public virtual Pageable<ChatMessageReadReceipt> GetReadReceipts(int? skip = null, CancellationToken cancellationToken = default)
```

Comment:
Same with this, do we need `= null` here?

Reply:
if we don't define default variables for the optional parameters, we are forcing the callers to provide them always.
i.e.
var clientCall = client.GetReadReceipts(null) instead of simply:
var clientCall = client.GetReadReceipts() instead of simply.